### PR TITLE
7219 move data dict out of downloads

### PR DESCRIPTION
**Pull request by brett-varney**
**High level description:**

Remove references to "Download Center" from the Data Dictionary page

**Technical details:**
This ended up being more than conditionally hiding the sidebar, so I updated the file structure and combined/simplified where I could.
- updated sitemap & route
- removed dd refs from download page
- moved dd files from download folders to their own
- renamed folders & files to remove casing (just in case)
- updated & simplified css
- updated internal links

**JIRA Ticket:**
[DEV-7219](https://federal-spending-transparency.atlassian.net/browse/DEV-7219)

**Mockup:**
https://bahdigital.invisionapp.com/share/S6IAFK24YKC#/screens

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge (Windows only)
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [x] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations [React Testing Library](react-testing-library.md)

Reviewer(s):
- [x] Design review complete
- [x] Code review complete

### CODE REVIEW

File: scripts/sitemaps/pages.js [hunk 1]
```diff
@@ -21,7 +21,8 @@ const routes = [
     '/state',
     '/recipient',
     '/disaster/covid-19',
-    '/disaster/covid-19/data-sources'
+    '/disaster/covid-19/data-sources',
+    '/data-dictionary'
```
Comment (MaxwellKendall) [inline scripts/sitemaps/pages.js:25]: nice

File: src/js/components/dataDictionary/DataDictionaryPage.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import MetaTags from 'components/sharedComponents/metaTags/MetaTags';
+import Header from "containers/shared/HeaderContainer";
+import Footer from "containers/Footer";
+import { getStickyBreakPointForSidebar } from 'helpers/stickyHeaderHelper';
+
+import DataDictionaryContainer from "containers/dataDictionary/DataDictionaryContainer";
+
+require('pages/dataDictionary/dataDictionaryPage.scss');
+
+export default class DataDictionaryPage extends React.Component {
```
Comment (MaxwellKendall) [inline src/js/components/dataDictionary/DataDictionaryPage.jsx:19]: Can we just make this a functional component?

File: src/js/containers/dataDictionary/DataDictionaryContainer.jsx [hunk 3]
```diff
@@ -67,27 +59,22 @@ export default class DataDictionaryContainer extends React.Component {
                 }, () => this.parseRows(content.rows));
             })
             .catch((err) => {
-                console.log(err);
+                console.error(err);
```
Comment (MaxwellKendall) [inline src/js/containers/dataDictionary/DataDictionaryContainer.jsx:62]: yes sir!!!

**Review by MaxwellKendall — CHANGES_REQUESTED** [review]
Very small change request WRT using a functional rather than class based component.

I wonder if it wouldn't be too much to ask or too much effort to add a loading spinner to the page when the table is loading?

Definitely a stretch goal for this but pulling the code down it seems in the worst case scenario, if the API request is taking a bit, we wont see a table for a second then it will just pop up under the search bar some what out of nowhere.

Also, probably a question for design, the section title and then the big title seems redundant? 
image

File: src/js/components/dataDictionary/DataDictionaryPage.jsx (hunk 2 above)
Comment (brett-varney) [reply src/js/components/dataDictionary/DataDictionaryPage.jsx:19]: Why? We have a mix so I did not know there was a standard, and not to argue, but no one has ever explained any functional advantages, and I see several advantages to class-based. Plus that would be another refactor to a working solution.

**Review by MaxwellKendall — APPROVED** [review]

File: src/js/components/dataDictionary/DataDictionaryPage.jsx (hunk 2 above)
Comment (MaxwellKendall) [reply src/js/components/dataDictionary/DataDictionaryPage.jsx:19]: You're right.

**Pull request merged**